### PR TITLE
Implement EZP-24999: As a bundle developer I want to use standard ezpublish semantic configuration

### PR DESCRIPTION
**Pull request by pspanja**
This PR resolves https://jira.ez.no/browse/EZP-24999

The goal is to be able to define semantic configuration and default settings from external bundle, that will be available under standard `ezpublish` configuration root and subject to resolving (ConfigResolver).

This is achieved through two new methods in Core bundle extension, intended to be called from bundle's `build()` method:

* `addConfigParser()`
* `addDefaultSettings()`

Default settings could probably work if imported into standard `ezpublish.yml`, however this approach does not require manual configuration.

### CODE REVIEW

**Comment by pspanja** [comment]
Ready for review, ping @andrerom @bdunogier @lolautruche

**Comment by lolautruche** [comment]
+1, but this must be documented in doc/specifications :wink: 
And this doc will then land to official doc site.

**Comment by bdunogier** [comment]
Cool one. Agree with @lolautruche about doc, it should be short, and it's an important one.

**Comment by pspanja** [comment]
Added specification in https://github.com/ezsystems/ezpublish-kernel/commit/fb2052c9bae8fcde3dfea5a88be36e35c97d4768.

File: doc/specifications/config/semantic_configuration_integration.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Integrating custom semantic configuration
+
+> Added in 6.0 / 2015.09
```
Comment (andrerom) [inline doc/specifications/config/semantic_configuration_integration.md]: 2015.09.02

**Comment by andrerom** [comment]
+1

File: doc/specifications/config/semantic_configuration_integration.md (hunk 1 above)
Comment (pspanja) [inline doc/specifications/config/semantic_configuration_integration.md]: Fixed and amended.

**Comment by lolautruche** [comment]
+1

**Pull request merged**